pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2ClientConfiguration.java

```diff
@@ -19,6 +22,9 @@
  * @since 1.7
  */
 public final class SAML2ClientConfiguration implements Cloneable {
+	private KeyStore keyStore;
+
+	private Resource keystoreResource;
 
     private String keystorePath;
```

Comment:
Mainly I wanted all your original code paths to still work so you could pass either the original keystorePath or the new keystoreResource. I figured someone in the wild may have made a call to the old methods and I didn't want my changes to break it. Is this not desirable?